packages/ses/src/tame-locale-methods.js

```diff
@@ -0,0 +1,54 @@
+import { assert } from './assert.js';
+import { getOwnPropertyNames, defineProperty } from './commons.js';
+
+const localePattern = /^(\w*[a-z])[L]ocale([A-Z]\w*)$/;
+
+// See https://tc39.es/ecma262/#sec-string.prototype.localecompare
+const nonLocaleCompare = function localeCompare(that) {
+  if (this === null || this === undefined) {
+    throw new TypeError(`this is not coercible`);
```

Comment:
Perhaps we can leave the debugger with a little more guidance, “Cannot localeCompare with null or undefined receiver”